sdks/go/test/run_validatesrunner_tests.sh

```diff
@@ -64,11 +64,11 @@ RUNNER=portable
 TIMEOUT=1h
 
 # Where to store integration test outputs.
-GCS_LOCATION=gs://temp-storage-for-end-to-end-tests
+GCS_LOCATION=gs://clouddfe-danoliveira/temp-storage-go-end-to-end-tests
```

Comment:
I think we want to revert these changes, and use the apache-beam-testing project instead.

Reply:
Whoops, forgot to exclude this file from the PR. Fixed.